### PR TITLE
DOCSP 33890 updating mongosync limitations bullets

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updating bullets for mongosync bullets

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-33890/reference/limitations/

## JIRA

https://jira.mongodb.org/browse/DOCSP-33890

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65491080bd8a7a95794e1687

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -135,9 +133,8 @@ Sharded Clusters
   syncing.
 - The shard key cannot be modified using :dbcommand:`reshardCollection`
   during syncing.
-- The maximum number of :ref:`shard key indexes
-  <sharding-shard-key-indexes>` is one lower than normal, 63 instead of
-  64.
+- The maximum number of indexes per sharded collection is 63, which is
+  one less than the :ref:`normal limit <limit-index-size>` of 64.
```
Comment (jason-price-mongodb) [inline source/reference/limitations.txt:137]: Per Sarah, consider "default" or "typical" rather than "normal". Understood not your original content :)

**Review by jason-price-mongodb — APPROVED** [review]
@ltran-mdb2 LGTM with suggestion. Cheers.

File: source/reference/limitations.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/reference/limitations.txt]: Done! Thanks for the feedback!

**Comment by ltran-mdb2** [comment]
Hi @ajayvijayakumar123 ! Could you give a tech review to this PR on updating mongosync limitations when you get a chance?

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -102,9 +98,11 @@ Sharded Clusters
 - Sync from a replica set to a sharded cluster has the following
   limitations:
 
-  - Collections included in the ``sharding.shardingEntries`` option for
-    the :ref:`c2c-api-start` command cannot be renamed to use a
-    different database while ``mongosync`` is running.
+  - MongoDB allows users to rename collections that the
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:101]:
```suggestion
  - mongosync allows users to rename collections that the
```

File: source/reference/limitations.txt [hunk 3]
```diff
@@ -135,9 +133,8 @@ Sharded Clusters
   syncing.
 - The shard key cannot be modified using :dbcommand:`reshardCollection`
   during syncing.
-- The maximum number of :ref:`shard key indexes
-  <sharding-shard-key-indexes>` is one lower than normal, 63 instead of
-  64.
+- The maximum number of indexes per sharded collection is 63, which is
+  one less than the :ref:`default limit <limit-index-size>` of 64.
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:137]: I think the hyperlink is incorrect. It should point to: https://www.mongodb.com/docs/upcoming/reference/limits/#mongodb-limit-Number-of-Indexes-per-Collection

**Review by ajayvijayakumar123 — CHANGES_REQUESTED** [review]
Looks great overall % 2 suggestions!

File: source/reference/limitations.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/reference/limitations.txt]: Done!

File: source/reference/limitations.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/reference/limitations.txt]: Done!

**Comment by ltran-mdb2** [comment]
Thanks for the feedback @ajayvijayakumar123 ! Could you take another look when you get a chance?

**Review by ajayvijayakumar123 — APPROVED** [review]
LGTM!

**Pull request closed without merging**